library/mps/writer.c

```diff
@@ -485,22 +515,32 @@ int mbedtls_writer_group_close( mbedtls_writer_ext *wr_ext )
 }
 
 int mbedtls_writer_attach( mbedtls_writer_ext *wr_ext,
-                           mbedtls_writer *wr )
+                           mbedtls_writer *wr,
+                           int pass )
 {
-    TRACE_INIT( "writer_check_attach" );
+    TRACE_INIT( "mbedtls_writer_attach" );
     if( wr_ext->wr != NULL )
         RETURN( MBEDTLS_ERR_WRITER_UNEXPECTED_OPERATION );
 
-    wr_ext->wr = wr;
+    wr_ext->passthrough = pass;
+    wr_ext->wr   = wr;
```

Comment:
This line is not aligned to anything.

Reply:
@AndrzejKurek Thanks for spotting, it used to be `wr_ext->pass`, but I forgot to adapt the alignmen when I renamed.

Reply:
Fixed.